1.data-exploration/scripts/nbconverted/sample_info_data_visualization.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+# import the necessary packages that will be utilized
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd 
+import pathlib
+import seaborn as sns
+import plotnine as gg
+from plotnine import *
+import warnings
+warnings.filterwarnings('ignore')
+
+
+# In[2]:
+
+
+# assign the desired file a variable using pathlib.Path command
+input_file = pathlib.Path("../0.data-download/data/sample_info.csv")
+
+# set the data frame to be the desired .csv file that is read by pandas(pd) using the pd.read_csv(desired file read as a previously defined variable)
+df = pd.read_csv(input_file)
+
+# assign the desired file a variable using pathlib.Path command
+input_file2 = pathlib.Path("../0.data-download/data/CRISPR_gene_dependency.csv")
+
+# set the data frame to be the desired .csv file that is read by pandas(pd) using the pd.read_csv(desired file read as a previously defined variable)
+df2 = pd.read_csv(input_file2)
```

Comment:
Also here:

```suggestion
dependency_df = pd.read_csv(input_file2)
```

(or `x_df`)